app/screens/StudyHelp/Commentary/index.js

```diff
@@ -120,23 +120,24 @@ const Commentary = (props) => {
             {props.parallelMetaData?.revision !== null &&
               props.parallelMetaData?.revision !== "" && (
                 <Text textBreakStrategy={"simple"} style={style.metadataText}>
-                  <Text>Copyright:</Text> {props.parallelMetaData?.revision}
-                </Text>
-              )}
-            {props.parallelMetaData?.copyrightHolder !== null &&
-              props.parallelMetaData?.copyrightHolder !== "" && (
-                <Text textBreakStrategy={"simple"} style={style.metadataText}>
-                  <Text>License:</Text>{" "}
-                  {props.parallelMetaData?.copyrightHolder}
+                  <Text style={style.footerText}>Copyright:</Text>{" "}
```

Comment:
Change Copyright to Publishing Year
Also add it in the metadata json in the fetchAllContent.js file